### PR TITLE
Fix failing guide tests (Kafka Java 17...was rabbitmq)

### LINKED ISSUE #1039: fix micronaut-rabbitmq-rpc tests

### PR DESCRIPTION
**Pull request by wetted**
fixes #1039

### CODE REVIEW

**Comment by wetted** [comment]
I don't know what the failures were that are referenced by #1039 but they seems to work OK now. However, other tests in guides are failing and need investigation. I'll create issues for those.

File: guides/micronaut-mongodb-asynchronous/metadata.json [hunk 1]
```diff
@@ -5,6 +5,8 @@
   "tags": ["mongodb"],
   "categories": ["Data Access"],
   "publicationDate": "2021-10-13",
+  "skipMavenTests": true,
+  "skipGradleTests": true,
```
Comment (timyates) [inline guides/micronaut-mongodb-asynchronous/metadata.json]: This passes locally for me

File: guides/micronaut-kafka/metadata.json [hunk 2]
```diff
@@ -5,6 +5,8 @@
   "tags": ["kafka", "messaging"],
   "categories": ["Messaging"],
   "publicationDate": "2021-07-13",
+  "skipMavenTests": true,
+  "skipGradleTests": true,
```
Comment (timyates) [inline guides/micronaut-kafka/metadata.json]: This passes locally for me

**Review by timyates — CHANGES_REQUESTED** [review]
All but the Oracle Cloud Streaming ones pass locally for me...

The oracle cloud streaming doesn't run on my arm laptop, so I can't verify it 😔. Did it fail on CI?

File: guides/micronaut-kafka/metadata.json (hunk 2 above)
Comment (wetted) [reply guides/micronaut-kafka/metadata.json]: The Kafka tests pass for me on Java 11, but fail on Java 17.

Comment (timyates) [reply guides/micronaut-kafka/metadata.json]: How are you testing it?

Running `./gradlew micronautKafkaBuild` it passes locally for me with both:

- Java 11 https://ge.micronaut.io/s/ehkkojjwayuue#infrastructure
- Java 17 https://ge.micronaut.io/s/cpsuba3ar3keo#infrastructure

Comment (wetted) [reply guides/micronaut-kafka/metadata.json]: @timyates I am running the same task as you. I'll double check it again.

**Pull request closed without merging**